src/main/kotlin/app/revanced/patches/youtube/layout/branding/header/CustomHeaderPatch.kt

```diff
@@ -48,8 +48,8 @@ object ChangeHeaderPatch : ResourcePatch() {
         ),
         title = "Header",
         description = """
-            The header to use in top bar or the path to a custom header.
-            The path to a folder containing one or more of the following folders matching the DPI of your device:
+            Can be a header name or a path to a custom header folder to use in the top bar.
+            A path to a folder must contain one or more of the following folders matching the DPI of your device:
```

Comment:
We refer to the specific path if used for the option.
```suggestion
            The path to a folder must contain one or more of the following folders matching the DPI of your device:
```